packages/server/lib/reporter.coffee

```diff
@@ -1,6 +1,8 @@
 _     = require("lodash")
 path  = require("path")
-Mocha = require("mocha")
+## mocha-v* is used to allow us to have later versions of mocha specified in devDependencies
+## and prevents accidently upgrading this one
+Mocha = require("mocha-v2")
```

Comment:
did cleanly upgrading the `server` to 7.0.1 cause failures here?

Reply:
yes, big time

Reply:
:scream: